src/ice/agent.rs

```diff
@@ -761,63 +761,82 @@ impl IceAgent {
     /// Tells whether the message is for this agent instance.
     ///
     /// This is used to multiplex multiple ice agents on a server sharing the same UDP socket.
-    /// For this to work, the server should operate in ice-lite mode and not initiate any
-    /// binding requests itself.
```

Comment:
This comment seemed incorrect to me, to my knowledge there's nothing about locally-originated binding requests that stop this method from working properly.

Reply:
Yeah. I don't know why I wrote that. Brainwaves…